### DISCUSSION TITLE
PyRoxy

### DISCUSSION THREAD

**Discussion by ehhtehh** [MHProDev/MHDDoS#437] [Q&A]
im getting this error..  File "start.py", line 28, in <module>
    from PyRoxy import Proxy, ProxyChecker, ProxyType, ProxyUtiles
ModuleNotFoundError: No module named 'PyRoxy'

any idea how to fix?

**Comment by AbdullahZafs** [comment]
Try the following code in your Terminal/Cmd

`pip3 install PyRoxy`

**Reply by imtiazfahim256** [reply]
still same issue
![Screenshot 2024-02-06 144909](https://github.com/MatrixTM/MHDDoS/assets/61643273/61829595-adab-4fcc-af5d-35f1262a9128)

**Reply by bl4ckr0ss3** [reply]
I get "Already satisfied.." from WSL2, any solution maybe after years ?

**Reply by Nour-Zaid** [reply]
run this command and you should be fine sudo pip3 install -r requirements.txt --break-system-packages --target=/usr/local/lib/python3.12/dist-packages
